src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -533,6 +536,25 @@ message RpcLog {
   RpcLogCategory log_category = 8;
 }
 
+// Used to send metrics back to the Host 
+message RpcMetric {
+  // Matching ILogger semantics
+  // https://github.com/aspnet/Logging/blob/9506ccc3f3491488fe88010ef8b9eb64594abf95/src/Microsoft.Extensions.Logging/Logger.cs
+  // Level for the Log
+  
+  // Unique id for invocation (if exists)
+  string invocation_id = 1;
+
+  // Name of the given metric
+  string name = 2;
+
+  // Value of the metric
+  double value = 3;
+
+  // json serialized property bag, or could use a type scheme like map<string, TypedData>
+  string properties = 4;
```

Comment:
I see we use `string` for RpcLog.properties above -- but it feels like `map<string, string>` makes more sense for this.